backend/business-partner-agent/src/main/resources/security-keycloak.yml

```diff
@@ -0,0 +1,35 @@
+micronaut:
+  security:
+    endpoints:
+      logout:
+        path: /logout
+        get-allowed: true
+    redirect:
+      unauthorized:
+        url: /oauth/login
+      login-failure: /oauth/login
+    token:
+      roles-name: ${BPA_KEYCLOAK_ROLES_NAME:roles}
+      name-key: ${BPA_KEYCLOAK_NAME_KEY:preferred_username}
+    oauth2:
+      default-provider: keycloak
+      openid:
+        logout-uri: /api/logout
+        end-session:
+          redirect-uri: ${BPA_KEYCLOAK_REDIRECT_URI:}
```

Comment:
As this is special magic and not part of micronaut I would give it an own namespace like `bpa.security.oauth2.end-session.keycloak.enabled` so that it becomes clear that this is something that was added on top.

Reply:
Have removed this entirely, using the @RequiresKeycloak, which reads the `micronaut.security.oauth2.clients.keycloak` property.